server/conf/application.conf

```diff
@@ -71,8 +71,6 @@ auth {
   # Configure which IDP to use for applicant and admin authentication.
   applicant_idp = "idcs"
   applicant_idp = ${?CIVIFORM_APPLICANT_IDP}
-  admin_idp = "adfs"
-  admin_idp = ${?CIVIFORM_ADMIN_IDP}
```

Comment:
Why is it safe to remove this line?

Reply:
We don't actually use it anywhere.  We only have one option for admin identity providers (adfs).  It was confusing/misleading to provide it as an option